primo/opt_model/model_options.py

```diff
@@ -322,3 +322,43 @@ def solve_model(self, **kwargs):
 
         # In all other cases, return the optimal campaign
         return self._opt_model.get_optimal_campaign()
+
+    def update_cluster(self, add_widget_return):
+        """
+        Updates the campaign candidates by changing the cluster numbers for specific wells.
+
+        Parameters
+        ---------
+        add_widget_return : AddWidgetReturn
+            An AdWidgetReturn object which includes information on wells selected to add
+            to the existing optimal P&A projects
+        """
+        existing_clusters = add_widget_return.existing_cluster
```

Comment:
Again minor, but if the return from the widget is expected to be a list with multiple items, should the object attribute be named existing_clusters instead of existing_cluster?